.github/workflows/build.yml

```
@@ -0,0 +1,112 @@
+          scons -j8 p=windows bits=64
+          ls ./bin
+      - name: Publish artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: godotjs.windows.tools.64.exe
+          path: /godot/bin/godot.windows.tools.64.exe
+  build_mac:
+    name: Build Mac
+    runs-on: macos-latest
+    steps:
+      - name: Get dependencies
+        run: |
+          brew install scons yasm
+      - name: Get repositories
+        # TODO: Migrate this to use checkout action for ECMAScript.git rather than clone
+        run: |
+          cd ~
+          git clone https://github.com/godotengine/godot.git 
+          cd ./godot
+          git checkout 3.2.2-stable
+          git clone https://github.com/GodotExplorer/ECMAScript.git ./modules/ECMAScript
+      - name: Build godot with ECMAScript module for MacOS
+        # TODO: Support apple silicon with 3.2.3 for universal binaries - https://docs.godotengine.org/en/stable/development/compiling/compiling_for_osx.html
+        run: | 
+          cd ~/godot
+          scons -j8 p=osx arch=x86_64
+          ls ./bin
+      - name: Publish artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: godotjs.osx.tools.x86_64
+          path: ~/godot/bin/godot.osx.tools.x86_64
+  build_html5:
+    name: Build HTML5 Exporter
+    runs-on: ubuntu-latest
+    container:
+      image: flux159/godot:1.0.0
+    steps:
+      - name: Get repositories
+        # TODO: Migrate this to use checkout action for ECMAScript.git rather than clone
+        # TODO: HTML5 builds may be broken with quickjs_debugger, so using checkout 01bc02
+        run: |
+          rm -rf /godot && cd /
+          git clone https://github.com/godotengine/godot.git 
+          cd ./godot
+          git checkout 3.2.2-stable
+          git clone https://github.com/GodotExplorer/ECMAScript.git ./modules/ECMAScript
+          cd modules/ECMAScript/ && git checkout 01bc02 && cd ../..
```

Same here just let the build fail